bmap/src/lib.rs

```diff
@@ -35,12 +37,14 @@ pub enum CopyError {
 pub fn copy<I, O>(input: &mut I, output: &mut O, map: &Bmap) -> Result<(), CopyError>
 where
     I: Read + SeekForward,
-    O: Write + SeekForward,
+    O: Write + SeekForward + AsRawFd,
```

Comment:
This is a too heavy requirement as it will limit where we can use the copy function; E.g. that blocks using it if the output is not something that can be a raw file (e.g. if it's a usb protocol where we write to some on-device flash)